README.md

```diff
@@ -14,6 +14,7 @@ The generated representation is described using a
 middleware information.
 
 Supported cloud middleware providers:
+
```

Comment:
yes, my comment goes in that line: change needs to be documented, then update the docs relative to that change, but do not change any other random bits because then it's really hard to isolate issues.

Reply:
this changes was due to linting error reported by the mdl markdown checker.
And I was willing to make the tests pass as I've been introducing linting.

Reply:
ok, my bad, was assuming you were trying to update docs. I'll go 🤐

Reply:
No problem.
In the end, I think we all agree on the need to have coherent/focus PR, so once the big glue stuff will be merged we should no more have such a gigantic merge.